docs/support_matrix.rst

```diff
@@ -0,0 +1,35 @@
+  | - Turing                         |            | see |compatibility link|_ | - RHEL 8                              |                         |                     |                    |                                                 |
+  | - Volta                          |            |                           | - and other |PEP599 link|_ compatible |                         |                     |                    |                                                 |
+  +----------------------------------+------------+---------------------------+---------------------------------------+-------------------------+---------------------+--------------------+-------------------------------------------------+
+  | - Turing                         | cuda100    | 10.0                      | - Ubuntu 14.04                        | SM 3.5 and later        | r410 or later       | linux x86_64       | - Prebuild wheel available from developers page |
+  | - Volta                          |            |                           | - Ubuntu 16.04                        |                         |                     |                    | - Conda can be build from source                |
+  | - Pascal                         |            |                           | - Ubuntu 18.04                        |                         |                     |                    |                                                 |
+  | - Maxwell                        |            |                           | - RHEL 7                              |                         |                     |                    |                                                 |
+  | - Kepler                         |            |                           | - and other |PEP599 link|_ compatible |                         |                     |                    |                                                 |
+  +----------------------------------+------------+---------------------------+---------------------------------------+-------------------------+---------------------+--------------------+-------------------------------------------------+
+  | - Xavier                         | NA         | 10.2                      | Jetpack 4.4                           | SM 5.3 and later        | Jetpack 4.4         | Jetpack 4.4        | - Python wheel can be build from source         |
+  +----------------------------------+------------+---------------------------+---------------------------------------+-------------------------+---------------------+--------------------+-------------------------------------------------+
+  | - Xavier                         | NA         | 10.2                      | DRIVE OS 5.1.0.0                      | SM 5.3 and later        | DRIVE OS 5.1.0.0    | DRIVE OS 5.1.0.0   | - Native library can be build from source       |
```

Comment:
`NA` might mean "Not available" or "Not applicable", which would mean 2 different things. IMHO we might put the whole word here

Reply:
Done